.github/workflows/dart.yml

```diff
@@ -19,12 +19,21 @@ jobs:
       uses: subosito/flutter-action@v2
       with:
         channel: 'stable'
+    
+    - name: pub get
+      run: flutter pub get
+    
+    - name: pub upgrade
+      run: flutter pub upgrade --major-versions
 
     - name: Lint
       run: dart format --output=none --set-exit-if-changed .
-
-    - name: Pub Upgrade
-      run: flutter pub upgrade
+    
+    - name: dart anaylze
+      run: dart analyze
+    
+    - name: flutter analyze
```

Comment:
@oxyfalcon Do we need both flutter and dart analyse commands. Shouldn't one be sufficient

Reply:
@bonamiAkash dart analyse is mainly for the packages and the flutter analyse is mainly for the app level checks
Please check this link in stackOverflow -> [https://stackoverflow.com/questions/65308789/why-is-flutter-analyze-different-from-dart-analyze](url)

Reply:
@oxyfalcon ok
I guess this is fine then.